### PR TITLE
Move heading above viewer and truncate title to one line

### PR DESCRIPTION
**Pull request by davidpmccormick**
Screenshot 2019-03-18 at 14 51 33

### CODE REVIEW

File: catalogue/webapp/pages/item.js [hunk 1]
```diff
@@ -293,6 +293,43 @@ const ItemPage = ({
...
+          className={classNames({
+            [spacing({ s: 4 }, { margin: ['bottom'] })]: true,
+            [spacing({ s: 6 }, { padding: ['top'] })]: true,
+          })}
+        >
+          <h1
+            className={classNames({
+              [font({ s: 'HNM3', m: 'HNM2', l: 'HNM1' })]: true,
+            })}
+            style={{
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/item.js]: A component called `SinglLineEllipsised` perhaps?
Just thinking about weening out the multiple ways of doing things inline styles being that.

File: catalogue/webapp/pages/item.js [hunk 2]
```diff
@@ -293,6 +293,43 @@ const ItemPage = ({
...
+              whiteSpace: 'nowrap',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+            }}
+            title={title}
+          >
+            {title}
+          </h1>
+          <NextLink
```
Comment (jamesgorrie) [inline catalogue/webapp/pages/item.js]: Why do we cut out the page and query when we go back?

File: catalogue/webapp/pages/item.js (hunk 1 above)
Comment (davidpmccormick) [reply catalogue/webapp/pages/item.js]: Yeah, good shout. I was thinking that this all felt like it was going to need so much design overhaul that it didn't warrant componentising, but a thing that truncates text to a single line couldn't hurt.

File: catalogue/webapp/pages/item.js (hunk 2 above)
Comment (davidpmccormick) [reply catalogue/webapp/pages/item.js]: Oversight. Will update.

**Pull request merged**